docs/developer-documentation/front-end-development.md

```diff
@@ -1,4 +1,4 @@
-# Browsertrix Cloud frontend
+# Front-End Development
```

Comment:
```suggestion
# Frontend Development
```

Nitpick: I believe it's more common/correct when using as a noun (e.g. "I'm working on the Browsertrix frontend" v.s. "I'm a front-end developer working on Browsertrix")